.github/workflows/daily-bundler.yml

```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   daily_bundler:
     runs-on: ubuntu-18.04
+    if: github.repository == "rubygems/rubygems"
```

Comment:
Can you also do this for `monthly-bundler.yml` ?

Reply:
Fixed.  Sorry, I missed the monthly...